Packages/Primitives/Sources/Extensions/BigInt+Primitives.swift

```diff
@@ -28,6 +28,11 @@ public extension BigInt {
         return BigInt(result.rounded())
     }
     
+    func multiply(byPercentage percentage: Double) -> BigInt {
+        let result = Double(self) * percentage
+        return BigInt(result.rounded())
```

Comment:
```suggestion
        return BigInt(result.rounded())
```

can we avoid rounding? when working with `BigInt` this could lead to bugs.